bpf/http_ssl.c

```diff
@@ -164,10 +164,11 @@ int BPF_UPROBE(uprobe_ssl_write, void *ssl, const void *buf, int num) {
     ssl_args_t args = {};
     args.buf = (u64)buf;
     args.ssl = (u64)ssl;
-    args.len_ptr = 0;
 
     bpf_map_update_elem(&active_ssl_write_args, &id, &args, BPF_ANY);
```

Comment:
We still must keep a map of ssl write arguments, since it's used to connect the SSL pointer to a an active connection, by the tcp_sendmsg kprobe.